syndesis/plugins/org.fusesource.ide.syndesis.extension.core/src/org/fusesource/ide/syndesis/extensions/core/util/SyndesisExtensionsUtil.java

```diff
@@ -31,11 +31,15 @@ public static boolean isValidSyndesisExtensionVersion(String version) {
 		if (parts.length<2 || version.trim().endsWith(".") || version.trim().startsWith(".")) {
 			valid = false;
 		} else {
+			int foundNumbers = 0;
 			for (String part : parts) {
 				try {
 					Integer.parseInt(part);
+					foundNumbers++;
```

Comment:
I don't get from the code why this foundNumbers has been introduced. I think adding a test might help to understand it and avoid regression